core/src/main/java/org/stellar/anchor/sep24/Sep24TransactionStore.java

```diff
@@ -60,4 +63,21 @@ List<Sep24Transaction> findTransactions(
    */
   @SuppressWarnings("UnusedReturnValue")
   Sep24Transaction save(Sep24Transaction sep24Transaction) throws SepException;
+
+  /**
+   * Finds multiple transactions that matches following criteria
+   *
+   * @param to upper date to search to
+   * @param from lower date to search from
+   * @param orderBy date type that used for the search
+   * @param limit limit
+   * @param offset offset
+   * @return list of transactions
+   */
+  List<? extends Sep24Transaction> findBulk(
```

Comment:
Should we use the `findTransactions()` function already defined instead of `findBulk()`?